### DISCUSSION TITLE
What happened with localization?

### DISCUSSION THREAD

**Discussion by WelterDevelopment** [microsoft/WindowsAppSDK#720] [Q&A]
Porting an UWP app to Project Reunion is a nightmare... my x:Uid xaml attributes are causing uncatchable app crashes..

Why is IntelliSense suggesting me 'x:Uid' if its not working?
Why is ye good old 'Strings/xx-XX/Ressources.resw' technique that was universally used in uwp and xamarin projects and that everybody is familiar with seemingly not supported in Project Reunion? Will it be supported in .NET MAUI? Or is the 'MRT Core' with its horrible documentation the long-term way to go?

**Comment by mrlacey** [comment]
@WelterDevelopment I've not had any problems with using `x:Uid` and .resw files with WinUI3 as long as the right content type is specified for the .resw files.
Have you got an example to reproduce the problems you're seeing?

**Comment by axelandrejs** [comment]
The same .resw localization works with Reunion. If you have a repro of it not working, can you share the project?

MRT core is a replacement for inbox MRT (Windows.ApplicationModel.Resources), and behaves mostly the same. If you have code that calls the resource manager directly (that is, not using x:Uid) and is difficult to port, can you share the code pattern.
